styles/style.css

```
@@ -1,10 +1,112 @@
-/*
-Colors to use:
-dark blue: #06365f
-light blue: #1c5380
-*/
 
-body {
+:root{
+  --dark-Blue: #06365f;
+  --light-Blue: #1c5380;
+  --color-white:#e2eaf0;}
+
+body{
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+header{
+  width: 90%;
+}
+footer{
+  width: 100%;
+ margin-left: 15%;
+}
+h1{
+  color: var(--dark-Blue);
+}
+ul li{ 
+  list-style-type: none;
+}
+ul a{
+  text-decoration: none;
+  font-weight: bold;
+ }
+.nav-bar{
+  width: 100%;
+  background-image: url(/images/bgtile.png);
```

Aquí tienes que salir de la carpeta con `..` para indicar que entre en ìmages`
```suggestion
  background-image: url(../images/bgtile.png);
```

---

Vlae gracias! Sigo entrenandome!